contribute-to-docs/getting-started/previewing-docs-locally.md

```diff
@@ -10,7 +10,11 @@ If you choose to run the site locally, we strongly recommend using a container.
 
 Regardless of the method used, when you submit a PR, a live preview link will be available in a comment on the PR.
 
-## Use the Docker container
+## (Option 1): Use the Docker container
+
+You can use [Docker Desktop](https://www.docker.com/products/docker-desktop) or any docker engine supported for your operating system that is compatible with the `docker` CLI, for example [colima](https://github.com/abiosoft/colima).
+
+### Live preview
 
 To start the live preview, run the following script.
```

Comment:
Do you have to be in a particular directory to run these scripts? If so, maybe mention navigating to that directory.

Reply:
I'm going to update the scripts that they can be run from any directory